stable/aws-vpc-cni/templates/eniconfig.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: crd.k8s.amazonaws.com/v1alpha1
 kind: ENIConfig
 metadata:
-  name: {{ required ".Values.eniConfig.region must be specified" $.Values.eniConfig.region }}{{ $key }}
```

Comment:
Why was this removed? This is a breaking change and doesn't seem to be documented anywhere?

Reply:
I opened https://github.com/aws/eks-charts/pull/739 with the intent to propose reverting this, as it seems to be an accidental change.